CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [#7663](https://github.com/rubocop-hq/rubocop/pull/7663): Add new `Style/HashTransformKeys` and `Style/HashTransformValues` cops. ([@djudd][], [@eugeneius][])
 * [#7619](https://github.com/rubocop-hq/rubocop/issues/7619): Support autocorrect of legacy cop names for `Migration/DepartmentName`. ([@koic][])
 * [#7659](https://github.com/rubocop-hq/rubocop/pull/7659): Layout/LineLength autocorrect now breaks up long lines with blocks. ([@maxh][])
+* Add `BracesRequiredMethods` parameter to `Style/BlockDelimiters` to require braces for specific methods such as Sorbet's `sig`. ([@maxh][])
```

Comment:
There is a mismatch between the `CHANGELOG` entry and the actual configuration option: `BracesRequiredMethods` vs. `RequiredBracesMethods`. 🙂

Reply:
Yikes :)

Standardized on `BracesRequiredMethods` throughout.